### PR TITLE
Remove deprecations marked for v5.1

### PR DESCRIPTION
**Pull request by timwoj**
This is a counterpart to https://github.com/zeek/zeek/pull/2195, and also shouldn't be merged until 5.0 is fully released to avoid possible merge conflicts with things we might need to cherry-pick from master to release-5.0.

This is the remainder of the deprecations that are marked for removal in 5.0. It should probably be reviewed commit-by-commit since each commit should be fairly self-contained and not touch other areas.

### CODE REVIEW

File: scripts/base/files/x509/__load__.zeek [hunk 1]
```diff
@@ -1,4 +1,4 @@
 @load ./main
 @load ./certificate-event-cache
 
-@load ./log-ocsp
+@load ./log-ocsp
```
Comment (0xxon) [inline scripts/base/files/x509/__load__.zeek:4]: This one is a mistake, right? The only difference is now that the EOL at the end of the file is missing - which should be there.

File: src/Dict.cc [hunk 2]
```diff
@@ -530,80 +328,82 @@ TEST_CASE("dict robust iteration replacement")
 	delete val4;
 	}
 
+// TODO: should this test be rewritten for the new iteration style?
+/*
 TEST_CASE("dict iterator invalidation")
-	{
-	PDict<uint32_t> dict;
+    {
+    PDict<uint32_t> dict;
```
Comment (0xxon) [inline src/Dict.cc:335]: These and the following changes all look extremely weird - I know that this is now commented out, but it simultaneously changes all indentation tabs to spaces.

That is not right, is it?

**Review by 0xxon — COMMENTED** [review]
Looks good to me, minus the indentation changes that I commented on.

That being said - I did not look at every single line in excessive detail :)

File: scripts/base/files/x509/__load__.zeek (hunk 1 above)
Comment (timwoj) [reply scripts/base/files/x509/__load__.zeek]: Fixed.

File: src/Dict.cc (hunk 2 above)
Comment (timwoj) [reply src/Dict.cc]: I went back and reimplemented that test using the new iterators, like I should have done in the first place.

**Pull request merged**